elastic/kibana
# Adding model versions for all remaining so types without model versions

Resolves https://github.com/elastic/kibana/issues/184618

## Summary

Adds v1 schemas for all remaining Response Ops owned saved object types:
* `connector_token`
* `api_key_pending_invalidation`
* `maintenance-window`
* `rules-settings`

## To Verify

1. Run ES and Kibana on `main` and create saved objects for each of the above types:
  a. Create an OAuth ServiceNow ITOM connector to create a `connector_token` saved object
  b. Create a rule, let it run, and then delete the rule. This will create an `api_key_pending_invalidation` SO and 2 `rules-settings` SOs
  c. Create some maintenance windows, both with and without filters
2. Keep ES running and switch to this branch and restart Kibana. Then verify you can read and modify the existing SOs with no errors
  a. Test the ServiceNow ITOM connector, which should read the `connector_token` SO
  b. Modify the rules settings and then run a rule to ensure they're loaded with no errors
  c. Load the maintenance window UI and edit a MW
